src/commands/sfpowerkit/org/orgwideemail/create.ts

```diff
@@ -0,0 +1,80 @@
+import { core, SfdxCommand, FlagsConfig, flags } from "@salesforce/command";
+
+// Initialize Messages with the current plugin directory
+core.Messages.importMessagesDirectory(__dirname);
+
+// Load the specific messages for this file. Messages from @salesforce/command, @salesforce/core,
+// or any library that is using the messages framework can also be loaded this way.
+const messages = core.Messages.loadMessages(
+  "sfpowerkit",
+  "orgwideemail_create"
+);
+
+export default class OrgWideEmail extends SfdxCommand {
+  public static description = messages.getMessage(
+    "orgWideEmailCreateCommandDescription"
+  );
+
+  public static examples = [
+    `$ sfdx ancforce:testdate:orgwideemail --username scratchOrg --address email_addres --displayname "Test Address" --allprofile
+  `
+  ];
+
+  protected static flagsConfig: FlagsConfig = {
+    address: flags.string({
+      char: "a",
+      description: messages.getMessage("orgWideEmailAddressDescription"),
+      required: false
+    }),
+    displayname: flags.string({
+      char: "n",
+      description: messages.getMessage("orgWideEmailDisplaynameDescription"),
+      required: false
+    }),
+    allprofile: flags.boolean({
+      char: "p",
+      description: messages.getMessage("orgWideEmailAllprofileDescription"),
+      required: false
+    })
+  };
+  protected static requiresUsername = true;
+
+
+  public async run(): Promise<any> {
+
+    const address: string = this.flags.address;
+    const displayname: string = this.flags.displayname;
+    var allprofile = this.flags.allprofile ? true : false;
+
+    var orgWideAddressObj = {
+      Address: address,
+      DisplayName: displayname,
+      IsAllowAllProfiles: allprofile
+    };
+
+    this.ux.log("Creating email " + orgWideAddressObj.Address);
+
+    let response = await this.org.getConnection().request({
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json"
+      },
+      url: "/services/data/v44.0/sobjects/OrgWideEmailAddress",
```

Comment:
Update the api version from the connection object like const apiversion = await this.org.getConnection().retrieveMaxApiVersion();